pkg/xds/bootstrap/server.go

```diff
@@ -28,7 +28,7 @@ func (b *BootstrapServer) Start(stop <-chan struct{}) error {
 	mux.HandleFunc("/bootstrap", b.handleBootstrapRequest)
 
 	bootstrapServer := &http.Server{
-		Addr:    fmt.Sprintf(":%d", b.Port),
+		Addr:    fmt.Sprintf("127.0.0.1:%d", b.Port),
```

Comment:
Please revert this back so the server can be accessed outside of the control plane machine.